src/coffee/product-type-import.coffee

```diff
@@ -0,0 +1,90 @@
+_ = require 'underscore'
+Promise = require 'bluebird'
+ProductTypeImport = require 'sphere-product-type-import'
+{ExtendedLogger, ProjectCredentialsConfig} = require 'sphere-node-utils'
+package_json = require '../package.json'
+
+class ProductTypeImporter
+
+  ###
+  Prepare for importing, initialize used modules
+  ###
+  init: (argv) ->
```

Comment:
Well, the async stuff shouldn't go in the contructor :)

I think just a function `run` or `import` is enough, you just do everything there.